internal/service/device.go

```diff
@@ -15,22 +15,22 @@ import (
 )
 
 type DeviceStore interface {
-	CreateDevice(ctx context.Context, orgId uuid.UUID, device *api.Device) (*api.Device, error)
-	ListDevices(ctx context.Context, orgId uuid.UUID, listParams ListParams) (*api.DeviceList, error)
-	GetDevice(ctx context.Context, orgId uuid.UUID, name string) (*api.Device, error)
-	CreateOrUpdateDevice(ctx context.Context, orgId uuid.UUID, device *api.Device) (*api.Device, bool, error)
-	UpdateDeviceStatus(ctx context.Context, orgId uuid.UUID, device *api.Device) (*api.Device, error)
-	DeleteDevices(ctx context.Context, orgId uuid.UUID) error
-	DeleteDevice(ctx context.Context, orgId uuid.UUID, name string) error
-	ListAllDevicesInternal(map[string]string) ([]model.Device, error)
-	UpdateDeviceInternal(device *model.Device) error
+	Create(ctx context.Context, orgId uuid.UUID, device *api.Device) (*api.Device, error)
+	List(ctx context.Context, orgId uuid.UUID, listParams ListParams) (*api.DeviceList, error)
+	Get(ctx context.Context, orgId uuid.UUID, name string) (*api.Device, error)
+	CreateOrUpdate(ctx context.Context, orgId uuid.UUID, device *api.Device) (*api.Device, bool, error)
+	UpdateStatus(ctx context.Context, orgId uuid.UUID, device *api.Device) (*api.Device, error)
+	DeleteAll(ctx context.Context, orgId uuid.UUID) error
+	Delete(ctx context.Context, orgId uuid.UUID, name string) error
+	ListIgnoreOrg(map[string]string) ([]model.Device, error)
```

Comment:
this still feels like something that should be external to the interface. Can't we unify List to provide a way to expose a way to do something like this.  List(ctx, listParams. WithOrg(false)) maybe sth like

```
List(ctx context.Context, orgId uuid.UUID, listParams ListParams, opts listOpts...) (*api.DeviceList, error)
```

Reply:
One main difference is that this returns model.Device and not api.Device